internal/cluster/distribution/eks/eksworkflow/activity_update_addon.go

```diff
@@ -54,7 +54,8 @@ type UpdateAddonActivityInput struct {
 
 // UpdateAddonActivityOutput holds the output data of the UpdateAddonActivityOutput
 type UpdateAddonActivityOutput struct {
-	UpdateID string
+	UpdateID          string
+	AddonNotInstalled bool
```

Comment:
**Recommended**: In the future I suggest using positive terms on boolean variables. It is common development practice as negatively named variables are more complex to resolve in mind when reading code. (`AddonNotInstalled := false ; ... ; if !AddonNotInstalled {` is 3 logical operations to resolve vs [opposite case]. `AddonInstalled := false ; ... ; if !AddonInstalled` is only 2).